test/trento/discovery/policies/cluster_policy_test.exs

```diff
@@ -2008,6 +2015,7 @@ defmodule Trento.Discovery.Policies.ClusterPolicyTest do
                   cib_last_written: "Thu Feb 23 15:59:56 2023",
                   details: %HanaClusterDetails{
                     architecture_type: HanaArchitectureType.classic(),
+                    hana_scenario: HanaScenario.unknown(),
```

Comment:
Why `unknonwn`? Scale out should be performance optimized.
You can hardcode the type in the policy for this scenario